pkg/common/util/util.go

```diff
@@ -18,8 +18,8 @@ import (
 	"fmt"
 	"time"
 
-	commonv1 "github.com/kubeflow/common/pkg/apis/common/v1"
-	commonutil "github.com/kubeflow/common/pkg/util"
+	commonv1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
+	commonutil "github.com/kubeflow/training-operator/pkg/util"
```

Comment:
Maybe, we need to sort utils out.
We can work on this at follow-up PRs.

Reply:
Yes. lets track this in a different PR. There are some code duplication as well which can be cleaned up in a different PR